support/modules/single-file

```diff
@@ -10,6 +10,17 @@ dest_dir_file="$FILES"/files/dest_dir
 filename_file="$FILES"/files/filename
 permissions_file="$FILES"/files/permissions
 
+safe_copy() {
+    if [ -n "$3" ]; then
+        echo "safe_copy can only handle one file copy at a time" 1>&2
+        exit 2
+    fi
+    cp -a "$1" "$2".tmp || return $?
```

Comment:
Also, could we use the long option in scripts? It'd just be a good convention to follow I think

Reply:
I somewhat agree, but not for very common options. In this particular case I think the original is actually better, because the full name `--archive`, does not really reveal what it does, and is even a bit misleading (a better name would be `--preserve-attributes` or something like that). I think in this case it's better to look it up.

Definitely for our various options in mender-artifact I agree though, long option over short option.